recipe/install-r-xgboost.bat

```diff
@@ -2,10 +2,14 @@
 mkdir "%SRC_DIR%"\build-r
 pushd "%SRC_DIR%"\build-r
 
+set PATH=%LIBRARY_PREFIX%\bin;%PATH%
+
 cmake -G "Ninja" ^
     -DCMAKE_BUILD_TYPE:STRING="Release" ^
     -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ^
     -DCMAKE_POSITION_INDEPENDENT_CODE:BOOL=ON ^
+    -DDLLTOOL_EXE=%BUILD_PREFIX%\Library\bin\%DLLTOOL% ^
```

Comment:
Also is this still needed? Would think `%BUILD_PREFIX%\Library\bin` is in `%PATH%`. So would think `% DLLTOOL%` would get picked up by the normal search process

Though please let me know if there is something more to this

Reply:
I believe this was manually added because `cmake` was hard-coding an executable name for %DLLTOOL% that didn't exist.  These issues seem to be the result of changes to executable names with the UCRT migration.  We can also try removing this to see if it's still needed.

Reply:
And if it is still needed, a code comment would be much appreciated